### ISSUE TITLE
0.9 compatibility

### ISSUE THREAD

**Issue by trasher** [flassave/galette-plugin-Passages_de_grades#12]
Plugin needs to be updated for Galette 0.9. I can help on that.

**Comment by florianbusi** [comment]
> 
> 
> Plugin needs to be updated for Galette 0.9. I can help on that.

Hi @trasher ! This project seems to be stopped :-/

**Comment by trasher** [comment]
@florianbusi indeeed, but I'm not the author of the plugin; I can propose some help but I'm not going to take the ownership.

**Comment by florianbusi** [comment]
Yes Johan, obviously 😉 you already do a lot with Galette, thank you very
very much for that!


Le 6 nov. 2018 08:40, "Johan Cwiklinski" <notifications@github.com> a
écrit :

@florianbusi <https://github.com/florianbusi> indeeed, but I'm not the
author of the plugin; I can propose some help but I'm not going to take the
ownership.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/galette-community/plugin-grades/issues/12#issuecomment-436158597>,
or mute the thread
